refinery/templates/tool_manager/vis-tool-user-not-allowed.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block title %}
+Visualization Error
```

Comment:
the formatting seems off. HTML -> 1 tab/ 2 space. Some blocks' interiors need indenting.